dockers/docker-syncd-rpc/Dockerfile

```diff
@@ -44,5 +44,6 @@ COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
+    && sysctl -w net.core.rmem_max=509430500 \
```

Comment:
What's wrong with this line? Can you please explain it further?

I don't see any rationale to put this line into the host-machine level (systemd). In this case we will have two base systems (for tests and for production).

Also I don't see any rationale to change this systemctl before the tests, then restart ptf_nn_agent, then change it back again and restart ptf_nn_agent again.